build.gradle

```diff
@@ -56,10 +56,13 @@ repositories {
 }
 
 dependencies {
-	minecraft "com.mojang:minecraft:19w04b"
-	mappings "net.fabricmc:yarn:19w04b.2"
+	minecraft "com.mojang:minecraft:19w05a"
+	mappings "net.fabricmc:yarn:19w05a.6"
 
+	// Minecraft's JAR uses these annotations
 	compile 'com.google.code.findbugs:jsr305:3.0.2'
```

Comment:
is there anything against making that an `api` depencency? since minecraft uses it already, why not make it available to the mod devs as well?

Reply:
The hopes for adopting checker-qual, which is more future-proof as Guava - which Mojang pulls JSR305 from - has also adopted it.

Reply:
in that cause it should be `implementation`.. or `api`
compile really just exists as a catchall now and for old buildscripts

Reply:
This might be looked into come new Loom